### PR TITLE
Python - online interpreters/compilters

### PR DESCRIPTION
**Pull request by chaanakyaaM**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

Closes #10

Online Interpreters/Compilers are helpful for the beginners to write and run code, and also for experimenting with the code without setting up the environment.
If the beginners had any issues in setting up the development environment then in use this online interpreters/compilers as alternative.

```
<table width="100%">
      <tr>
        <th>Channel Name</th>
        <th>Description</th>
      </tr>
       <tr>
        <td><a href="https://www.online-python.com/">Online-python.com </a></td>
        <td>This website is a web-based integrated development environment (IDE) for Python programming. It allows beginners to write, run, and share Python code directly in the browser without needing to set up a local environment.</td>
      </tr>
      <tr>
        <td><a href="https://www.programiz.com/python-programming/online-compiler/">programiz.com</a></td>
        <td>This website serves as Online Python Compiler, it is a web-based tool for writing, running, and sharing Python code. It features an interactive editor that supports Python 3, allowing beginners to write code, execute it, and see output in real-time.</td>
      </tr>
      <tr>
        <td><a href="https://domsignal.com/python-online-compiler">domsignal.com</a></td>
        <td>The Domsignal Online Python Compiler is a user-friendly tool for writing, running, and debugging Python code directly in your browser. It supports Python 3.8.1 and provides an intuitive interface for code execution and output display</td>
      </tr>
      <tr>
        <td><a href="https://www.onlinegdb.com/online_python_compiler">onlinegdb.com</a></td>
        <td>The OnlineGDB Python Compiler is a versatile web-based tool that allows users to write, compile, run, and debug Python code. It features an interactive editor with various customization options, including different themes and editor modes (Normal, Emacs, Vim). </td>
      </tr>
      <tr>
        <td><a href="https://replit.com/languages/python3">replit.com</a></td>
        <td>This Replit's Online Python Compiler is a collaborative tool that allows users to write, run, and share Python code directly in their browser. It features real-time collaboration, making it ideal for teamwork and learning environments. Users can also deploy their projects easily</td>
      </tr>
</table>
```

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

> fill the bracket with x. Something like this [x]

### CODE REVIEW

File: Programming Languages/Python/readme.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Python development refers to creating applications, scripts, and software soluti
 - [Books](#books)<br>
 - [Youtube Channels](#youtube-channels)<br>
 - [YouTube Playlists](#youtube-playlist)
+- [Online Interpreters/Compilers](#online-interpreters)
```
Comment (jfmartinz) [inline Programming Languages/Python/readme.md:24]: Can you moved this under the [Tools](https://github.com/jfmartinz/ResourceHub/blob/efa41d174eb8a11459c1e85bcf283e46c8add7b7/Programming%20Languages/Python/readme.md#tools) Section as a child. Thanks

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by chaanakyaaM** [comment]
Done !

**Comment by chaanakyaaM** [comment]
Done !

**Review by jfmartinz — APPROVED** [review]
Thanks

**Pull request merged**